client/model/attributes_test.go

```diff
@@ -357,6 +357,16 @@ func TestAttributes(t *testing.T) {
 				expectPartVolume("piano", 0.82),
 			},
 		},
+		scoreUpdateTestCase{
+			label: "set volume using dynamic marking",
+			updates: []ScoreUpdate{
+				PartDeclaration{Names: []string{"piano"}},
+				AttributeUpdate{PartUpdate: DynamicMarking{Marking: "ppp"}},
+			},
+			expectations: []scoreUpdateExpectation{
+				expectPartVolume("piano", 0.23683),
```

Comment:
I think it might be good to define this in terms of the DynamicVolumes map, just so that if we ever update it, we don't have to do it in two places:

```suggestion
				expectPartVolume("piano", DynamicVolumes["ppp"]),
```

Reply:
Changed this. Also ran the tests and realized there's a test for "initial volume" equal to 1.0 that fails. Not sure how I missed that. I updated that to use `DynamicVolumes["mf"]`.